_component_design/option-tiles.md

```diff
@@ -0,0 +1,338 @@
+- **Icon (optional)**: The icon visually represents the option you can select with the option tile.
+- **Option label (required)**: The option label is the title of the option the user can select.
+- **Description (optional)**: A description briefly explains the option tile selection to a user.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Option tile composition"] [class:"image bg-light border"] %}
+<embed src="{{site.baseurl}}/assets/images/components/inputs-and-controls/option-tiles/optiontiles-composition.png" width="356"/>
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Style
+
+Use the following specifications when you construct option tiles.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Option tile style"] [class:"image bg-light border"] %}
+<embed src="{{site.baseurl}}/assets/images/components/inputs-and-controls/option-tiles/Optiontiles-style.png" width="411"/>
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Spacing
+
+Use these spacing guidelines when you construct option tiles.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Option tile spacing"] [class:"image bg-light border"] %}
+<embed src="{{site.baseurl}}/assets/images/components/inputs-and-controls/option-tiles/optiontiles-spacing.png" width="604"/>
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+### Behaviors
+
+Ensure that you construct option tiles so that they conform to the following single- and multi-select behaviors.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+#### Single-select interactions
+
+A single-select interaction is analogous to radios where the user can only select one option tile at a time.
+
+In the default state, the user can click any of the option tiles. You can construct option tiles so that one of the options is selected by default. The user can then make another selection, if necessary. When the user selects an option, the previously selected option tile becomes deselected and all remaining tiles change to the subdued state. The subdued state reinforces that only one option tile can be selected.
```

Comment:
Should we break this up into bullets? it feels hard to parse and totally liable to be skipped.

Reply:
I shortened this text quite a bit, by about 1/2 I think. See how that works for you now. I agree, it was a bit of a mouthful to get through! :)